lib/rubocop/cop/style/space_after_not.rb

```diff
@@ -16,10 +16,11 @@ class SpaceAfterNot < Cop
         MSG = 'Do not leave space between `!` and its argument.'.freeze
 
         def on_send(node)
-          receiver, method_name, *_args = *node
+          receiver, method_name, *args = *node
 
           return unless method_name == :!
           return unless receiver.loc.column - node.loc.column > 1
+          return if node.loc.selector.is?('not'.freeze) && args.empty?
```

Comment:
Hmm, I wonder why we added the `args.empty?` check. It's not like you can define a method called `not`...